sdk/azidentity/interactive_browser_credential.go

```diff
@@ -30,6 +30,9 @@ type InteractiveBrowserCredentialOptions struct {
 	// only when setting ClientID, and must match a redirect URI in the application's registration.
 	// Applications which have registered "http://localhost" as a redirect URI need not set this option.
 	RedirectURL string
+
+	// DisableInstanceDiscovery allows disconnected cloud solutions to skip instance discovery for unknown authority hosts.
+	DisableInstanceDiscovery bool
```

Comment:
Can you please place this after `ClientID`? The struct definition appears verbatim in the package docs; alphabetizing the fields makes them easier to scan.